### ISSUE TITLE
Bilder von der Festplatte

### ISSUE THREAD

**Issue by S1ngularity96** [mooxl/astaRedesign#8]
Die hochgeladenen Bilder aus den Blogs müssen wieder von der Festplatte geladen werden können, um über das Template eingebunden werden zu können. 

- Von der Anwendung kommt eine Anfrage für ein bestimmtes Bild, und der Server liefert dann die Datei zurück

Eine mögliche Idee dazu : 

- Den Pfad der gespeicherten Datei einfach in das "src" des <image> Tags einzubinden?
@mooxl siehe Frage...

**Comment by mooxl** [comment]
Genau! Dadurch, dass wir immer den selben Pfad haben und sich nur der Ordnername (was wiederum die ID der Bilder entpricht), können wir den Pfad hardcoden und den GET-Parameter (oder POST?) von der URL in den Pfad einpflegen.

**Comment by S1ngularity96** [comment]
ja, gut. Dann mache ich das so.
